### ISSUE TITLE
Sort Order News Items

### ISSUE THREAD

**Issue by freinhard** [OneGov/onegov.town#218]
> Das Verhalten bei Aktuelles ist ja so, dass diese nach dem Zeitpunkt der Erfassung sortiert werden.
> Richtig oder?
>  
> Wir hatten nun den Umstand, dass ein Eintrag vorerfasst wurde und erst später zur Veröffentlichung freigegen wurde. Diese Information hätte aber in der Reihenfolge zuerst sein sollen, die nicht dem Zeitpunkt der Erfassung entsprach.
>  
> Wäre es möglich, dass zu gegebener Zeit das so geändert werden könnt, dass nicht das Erstellungsdatum sondern ein Datum der Änderung genommen würde?
> Wobei auch das Flag entfernen als Update gelten sollte.

**Comment by href** [comment]
Since the modified date is a poor replacement for this (as news from way back could bubble up), we need to implement some kind of publication date if we want to go ahead with this.

**Comment by freinhard** [comment]
+1 for the publication date. 
This could go into the direction of a publication workflow which we want to avoid... We have to think about this.

**Comment by href** [comment]
Yeah, though it doesn't necessarily have to be an explicit publication date. We can also set it automatically upon publication. We just need it to be a separate entity.
